src/main/java/eu/enmeshed/model/relationships/Relationship.java

```diff
@@ -17,27 +18,24 @@
 @Setter
 @Builder
 public class Relationship implements WebhookData {
-
-  private String id;
-
-  private RelationshipTemplate template;
-
-  private Status status;
-
-  private String peer;
-
-  private IdentityInfo peerIdentity;
-
-  private List<RelationshipChange> changes;
-
-  public enum Status {
-    @JsonProperty("Pending")
-    PENDING,
-
-    @JsonProperty("Active")
-    ACTIVE,
-
-    @JsonProperty("Rejected")
-    REJECTED
-  }
+    private String id;
+    private RelationshipTemplate template;
+    private RelationshipStatus status;
+    private String peer;
+    private IdentityInfo peerIdentity;
+    private RelationshipCreationContent creationContent;
+    private List<RelationshipAuditLogEntry> auditLog;
+    private PeerDeletionInfo peerDeletionInfo;
+
+    @Data
+    public static class PeerDeletionInfo {
+        PeerDeletionStatus peerDeletionStatus;
+    }
+
+    public enum PeerDeletionStatus {
+        @JsonProperty("ToBeDeleted")
+        TO_BE_DELETED,
+        @JsonProperty("Deleted")
+        DELETED
+    }
```

Comment:
following your pattern `PeerDeletionStatus` should be inside the class `PeerDeletionInfo` and only be called Status

Reply:
Done

Reply:
please re-read my whole comment, you missed an important part.

Reply:
didn't get it, could you please explain?

Reply:
> following your pattern PeerDeletionStatus should be inside the class PeerDeletionInfo

done

> and only be called Status

not done

Reply:
Renamed